traffic_ops/testing/api/v3/deliveryservices_test.go

```diff
@@ -396,6 +397,31 @@ func GetTestDeliveryServices(t *testing.T) {
 	}
 }
 
+func GetInactiveTestDeliveryServices(t *testing.T) {
+	actualDSes, _, err := TOSession.GetDeliveryServicesV30WithHdr(nil, nil)
+	if err != nil {
+		t.Errorf("cannot GET DeliveryServices: %v - %v", err, actualDSes)
+	}
+	totalDSes := len(actualDSes)
+	params := url.Values{}
+	params.Set("active", strconv.FormatBool(false))
+	inactiveDSes, _, err := TOSession.GetDeliveryServicesV30WithHdr(nil, params)
+	if err != nil {
+		t.Errorf("cannot GET DeliveryServices: %v - %v", err, actualDSes)
+	}
+	if len(inactiveDSes) != 0 {
+		t.Errorf("expected none of the delivery services to be inactive, but got %v inactive", len(inactiveDSes))
```

Comment:
We shouldn't hardcode data assumptions like this in the tests. Someone might need to add a test in the future that would require an inactive DS, and this test would fail for no reason. Ideally, we should add at least one `active: false` delivery service to tc-fixtures.json, then iterate over all returned DSes and check whether they are all inactive or active, based on the query parameter that was used.